litellm/model_prices_and_context_window_backup.json

```diff
@@ -11163,13 +11163,71 @@
         "litellm_provider": "perplexity",
         "mode": "chat"
     },
+    "perplexity/sonar": {
+        "max_tokens": 128000,
+        "max_input_tokens": 128000,
+        "input_cost_per_token": 1e-6,
+        "output_cost_per_token": 1e-6,
+        "litellm_provider": "perplexity",
+        "mode": "chat",
+        "search_context_cost_per_query": {
+            "search_context_size_low": 5e-3,
```

Comment:
where did you get this from ? Can you add source

Reply:
https://docs.perplexity.ai/guides/pricing